examples/cluster-dns/dns-backend-rc.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
         - name: dns-backend
-          image: ddysher/dns-backend
+          image: gcr.io/google_samples/dns-backend:v1
```

Comment:
change this to `gcr.io/google_containers/example-dns-backend:v1` this image now exists.